cysimdjson/cysimdjson.pyx

```diff
@@ -152,6 +159,26 @@ cdef class JSONElement:
 		return ok == 0
 
 
+	def __iter__(self):
+
+		for _key in self.keys():
+			yield _key.decode('utf-8')
```

Comment:
This decoding needs to happen inside of the Cython code, not here ... for systematic reasons.

BTW: Can JSON dict has different key type to string?

Reply:
Pure JSON cannot: https://www.w3schools.com/js/js_json_syntax.asp